README-bb2-dev.md

```diff
@@ -30,4 +30,33 @@ Note: You may need to clean up already existing Docker containers, if you are ha
 
 ## Visual trouble shoot
 
-Install VNC viewer and point browser to http://localhost:5900 to monitor web UI interactions
\ No newline at end of file
+Install VNC viewer and point browser to http://localhost:5900 to monitor web UI interactions
+
+## Installing cms-bluebutton-sdk from test.pypi.org
+
+The package can be installed from the test instance of the PyPI site for development testing. The default behavior is to install from the main site.
+
+To utilize the test.pypi.org repository use the following commands:
+
+To create and start the containers:
+
+```
+INSTALL_FROM_TEST_PYPI="True" docker-compose up -d
```

Comment:
@JFU-GIT @ajshred  Do you think adding this "INSTALL_FROM_TEST_PYPI" build argument would be good to add to the Node sample app as well?

Reply:
> INSTALL_FROM_TEST_PYPI

@JFU-GIT @ajshred   If we wanted to do something like this for the Node sample app, wondering if a better naming would be "BUILD_DEVELOPMENT"?

If you think this is better, I can change the argument name?

Reply:
I think it's good since both registries provide similar mechanism to let dev exercise publish operation and test with it.

Reply:
Yes I would do it for both.  BUILD_DEVELOPMENT is a good name.